### PR TITLE
with-facebook-pixel: new implementation with app folder

### PR DESCRIPTION
**Pull request by niko20**
### What?

* **Facebook pixel implementation with the new `app folder` structure.**
* The pixel script can now be kept in the public folder.
* The Facebook script can be placed in the `layout.js` file
* A component called `FacebookPixel` is provided to handle the initial script setup and manage `PageView` events when the page changes.
* Classic implementation with `pages` folder now is in `_pages`

### Why?
To provide an example of implementing the new `app folder` structure with Next.js ˆ13, which includes using the new files like `layout`, `page`, etc.

### How?

Utilizing the `next/script` package and the `layout.js` file to initialize the Facebook pixel when the application starts and manage subsequent events.

### Evidence of Implementation:

Here are some examples that demonstrate the successful implementation of the Facebook pixel using the new `app folder` structure and `Next.js ˆ13`

Tracking main page `app/page.js`
image

image

Tracking another path `app/about/page.js`
image

image

navigation between pages with `next/link`
image

### CODE REVIEW

**Comment by niko20** [comment]
@lfades ✋🏻  please help me with this contribution, I worked on the last examples and now it's an example of Facebook pixel but using the new `app folder`.

😃

**Comment by dvakatsiienko** [comment]
Hi! The linter fails [here](https://github.com/vercel/next.js/pull/49880/files#diff-5ff9ad430ccabd722a96f5dde51ed416d0508f53bbe612bfd3225a588c165e13R3)

**Comment by niko20** [comment]
@dvakatsiienko Hi, I appreciate your support. The issue was "prettier"... thank you.

File: examples/with-facebook-pixel/app/page.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import Link from 'next/link'
+
+export default function Home() {
+  return (
+    <>
+      <h2>
+        Go to `app/layout.js` to see how implement Facebook Pixel in NextJS ˆ13
+        with app folder
```
Comment (leerob) [inline examples/with-facebook-pixel/app/page.js:8]:
```suggestion
        Go to `app/layout.js` to see how to implement Facebook Pixel in Next.js 13
        with the App Router
```

File: examples/with-facebook-pixel/app/readme.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+Implementation only for Next 13 version or greater with the app folder
```
Comment (leerob) [inline examples/with-facebook-pixel/app/readme.txt:1]:
```suggestion
Implementation for Next 13+ using the App Router.
```

File: examples/with-facebook-pixel/next.config.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+const nextConfig = {
+  experimental: {
+    appDir: true,
```
Comment (leerob) [inline examples/with-facebook-pixel/next.config.js:3]: This file can be removed

**Comment by ijjk** [comment]
Allow CI Workflow Run

- [ ] approve CI run for commit: d3cdd85118c1ea80f3c17c622044d4fafef508b1

Note: this should only be enabled once the PR is ready to go and can only be enabled by a maintainer

File: examples/with-facebook-pixel/next.config.js (hunk 3 above)
Comment (niko20) [reply examples/with-facebook-pixel/next.config.js]: @leerob  the suggested changes have been made. I appreciate your feedback and support. 🙂

**Review by leerob — APPROVED** [review]

**Pull request merged**